Rect42.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:math';
+
+import 'package:flutter/material.dart';
+
+class Rect42 extends StatefulWidget {

```

Comment:
you don't need it to be stateful

Reply:
and what rect 42 means?

Reply:
I have changed the class to stateless in the second pull request I have uploaded.

rect 42 is just the second version of the rect 40 class, I wanted to keep the old version as well.